jetson/include/core/cluster.h

```diff
@@ -0,0 +1,15 @@
+#ifndef CLUSTER_H
+#define CLUSTER_H
+
+#include <vector>
+
+// TODO: refactor to have a 2D point and a 3D point in a seperate file
```

Comment:
can you use the point class in `jetson/core/point` might need to generalize to 2D

Reply:
I'll do this as part of coordinate transform